src/FileServiceManager/FileService/Helpers/ImageHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using System.IO;
+
+namespace FileService.Helpers
+{
+    public static class ImageHelper
+    {
+        public static bool CheckExtension(string extension, IEnumerable<string> allowedExtensions)
```

Comment:
According to this article https://docs.microsoft.com/en-us/aspnet/core/mvc/models/file-uploads?view=aspnetcore-5.0#upload-small-files-with-buffered-model-binding-to-physical-storage it seems a good practice to allow only approved file extensions for the app's design specification and check the size of an uploaded file. Checking file extension gives us ability to restrict non-wanted extensions. It will be better to check file signature from byte array though, it could be implemented in the future.

Reply:
I mean what is the meaning to have a method which in turn calls another one method ? You can call directly that method

Reply:
I modified this method to check file signature instead. It could be done by comparing heading bytes of file